packages/rum-core/src/rumEvent.types.ts

```diff
@@ -987,6 +987,20 @@ export type RumViewEvent = CommonProperties &
         readonly count: number
         [k: string]: unknown
       }
+      /**
+       * List of slow frames during the view’s lifetime
+       */
+      readonly slow_frames?: {
+        /**
+         * Duration in ns between start of the view and the start of the slow frame
+         */
+        readonly start: number
+        /**
+         * Duration in ns of the slow frame
+         */
+        readonly duration: number
+        [k: string]: unknown
+      }[]
```

Comment:
@amortemousque not sure what I should do about this ? It was synced when I did `yarn rum-events-format:sync` IIRC.

Reply:
This is fine you can leave it.